src/battle/area_kmr_part_1/battles.c

```diff
@@ -6,41 +6,41 @@
 #include "battle/actor/spiked_goomba.h"
 
 Formation formation0 = {
-    { goomba, .position = 1, .priority = 10 },
+    { &goomba, .position = 1, .priority = 10 },
 };
 
 Formation formation1 = {
-    { goomba, .position = 1, .priority = 10 },
-    { goomba, .position = 2, .priority = 9 },
+    { &goomba, .position = 1, .priority = 10 },
+    { &goomba, .position = 2, .priority = 9 },
 };
 
 Formation formation2 = {
-    { goomba, .position = 0, .priority = 10 },
-    { goomba, .position = 1, .priority = 9 },
-    { goomba, .position = 2, .priority = 8 },
+    { &goomba, .position = 0, .priority = 10 },
+    { &goomba, .position = 1, .priority = 9 },
+    { &goomba, .position = 2, .priority = 8 },
 };
 
 Formation formation3 = {
-    { goomba, .position = 1, .priority = 10 },
+    { &goomba, .position = 1, .priority = 10 },
     { paragoomba, .position = 6, .priority = 9 },
```

Comment:
this is the first one I noticed this on, but there are others; why do only the goomba's have the & on them?

Reply:
hm. In that case, can you extern them properly, so this is consistent?